### PR TITLE
fix: Update `axios`, `@actions/github` and `dompurify`  [1.23.X]

### PR DESCRIPTION
**Pull request by Saibamen**
`11 vulnerabilities (1 low, 4 moderate, 5 high, 1 critical)`

EDIT:
`13 vulnerabilities (1 low, 6 moderate, 5 high, 1 critical)`

instead of

`14 vulnerabilities (1 low, 7 moderate, 5 high, 1 critical)`

Replaces PR https://github.com/louislam/uptime-kuma/pull/4647

Other vulnerabilities (and critical 9.8 CVSS https://github.com/advisories/GHSA-fqg8-vfv7-8fj8) could be fixed by simply deleting `package-lock.json` and doing `npm update`, but this is too much for PR. This was done by @louislam many times in the past...

Vite should be updated to `4.5.3` to fix `fs.deny` high vulnerability (7.5/10 CVSS score) (currently we are using `4.4.12` in 1.23.X branch).

### CODE REVIEW

**Review by CommanderStorm — CHANGES_REQUESTED** [review]
I don't quite get why https://github.com/louislam/uptime-kuma/pull/4647 was closed, anyway.

These are the relevant parts of https://github.com/louislam/uptime-kuma/pull/4647#pullrequestreview-1978218721

File: package.json [hunk 1]
```diff
@@ -82,8 +82,8 @@
         "@louislam/ping": "~0.4.4-mod.1",
         "@louislam/sqlite3": "15.1.6",
         "args-parser": "~1.3.0",
-        "axios": "~0.27.0",
-        "axios-ntlm": "1.3.0",
+        "axios": "~0.28.1",
```
[the remark this answers is not in the document]
Comment (Saibamen) [reply package.json:85]: This is diff: https://www.diffchecker.com/bIToqvVl/

It is better - added stack trace and `cause`

**Comment by Zaid-maker** [comment]
> simply deleting package-lock.json

What u mean by deleting `package-lock.json`??

**Comment by Saibamen** [comment]
> What u mean by deleting `package-lock.json`??

`del package-lock.json` and `npm install` to recreate `package-lock.json` file with latest packages in version ranges from `package.json`

**Comment by Zaid-maker** [comment]
> > What u mean by deleting `package-lock.json`??
> 
> `del package-lock.json` and `npm install` to recreate `package-lock.json` file with latest packages in version ranges from `package.json`

Ohh for recreating got it

**Pull request merged**